marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -61,6 +61,9 @@ const StepDefinitionTypeTitles = {
 };
 
 const Flows: React.FC<Props> = (props) => {
+  const storage = getViewSettings();
+  const storedFlows = storage?.run?.storedFlows;
```

Comment:
Can we use an "openFlows" key? "stored" doesn't add any value and "open" describes the UI state.

Reply:
Yeah, I like that better too.